osu.Game/Overlays/Chat/Tabs/ChannelSelectorTabChannel.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Online.Chat;
+
+namespace osu.Game.Overlays.Chat.Tabs
+{
+    public class ChannelSelectorTabChannel : Channel
```

Comment:
might feel better as a nested public class inside `ChannelSelectorTabItem` (since it's only really used there).